src/Components/Pronunciation/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { notification } from "antd";
+import { AiTwotoneSound } from "react-icons/ai";
+import "./Pronunciation.scss";
+
+const Pronunciation = ({ data }) => {
+  const getAudioPath = (data) => {
+    if (data.audio.length > 0) {
+      if (data.audio[0]) {
+        return data.audio[0];
+      } else if (data.audio.length > 1) {
+        return data.audio[1];
+      }
+    } else {
+      notification.error({
+        message: "Error",
+        description: "Pronunciation not found!",
+      });
```

Comment:
It's better to show the button only when pronunciations are available, rather than disappointing the user after allowing them to click on it.